src/components/sbb-pearl-chain-item/sbb-pearl-chain-item.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, h, Host, Prop } from '@stencil/core';
+import { PearlChainItemAttributes } from './sbb-pearl-chain-item.custom';
+
+@Component({
+  shadow: true,
+  styleUrl: 'sbb-pearl-chain-item.scss',
+  tag: 'sbb-pearl-chain-item',
```

Comment:
should this component not be called sbb-pearl-chain-vertical-item? Or can this be used also with other pearl chains?